dartsim/include/ignition/physics/dartsim/World.hh

```diff
@@ -45,13 +45,15 @@ class RetrieveWorld : public virtual Feature
 };
 
 /////////////////////////////////////////////////
+//! [feature template]
```

Comment:
Are these comments meant for the code snippets or is this some special doxygen command?

Reply:
Yeah this is to mark out which part of the file to put in a snippet. For example, the usage in the .md file is
```
\snippet dartsim/include/ignition/physics/dartsim/World.hh feature template
```
In the places that work correctly, the snippet would show up. Like this one in `07-implementing-a-physics-plugin.md`, which references a file in `examples` that match the `.cc` filter:
```
\snippet examples/hello_world_plugin/HelloWorldPlugin.cc include statements
```

Reply:
Ah, got it. Thanks!